### PR TITLE
docs: add blobstream example

### PR DESCRIPTION
**Pull request by jcstein**
## Overview
Screenshot 2024-04-18 at 11 31 17 AM

This PR also fixes a link that I gave @rach-id the wrong path for 🙈 

## Summary by CodeRabbit

- **New Features**
	- Added an example implementation of Blobstream proofs by CryptoKass with a GitHub repository link in the sidebar navigation.
  
- **Documentation**
	- Updated a hyperlink in the documentation related to building blobstream rollups to point to the correct location.

### CODE REVIEW

File: .vitepress/config.ts [hunk 1]
```diff
@@ -696,6 +696,10 @@ function sidebarHome() {
               text: "Querying the Blobstream proofs",
               link: "/developers/blobstream-proof-queries",
             },
+            {
+              text: "Example implementation of Blobstream by CryptoKass",
```
Comment (rach-id) [inline .vitepress/config.ts:700]: Should ask for his permission first?

**Review by rach-id — COMMENTED** [review]
:shipit:

File: .vitepress/config.ts (hunk 1 above)
Comment (jcstein) [reply .vitepress/config.ts:700]: @CryptoKass do you mind if we add this? do yo have any suggested changes for this?

Comment (rach-id) [inline .vitepress/config.ts:700]:
```suggestion
              text: "Example implementation of Blobstream proofs by CryptoKass",
```

Comment (CryptoKass) [reply .vitepress/config.ts]: This looks good 👍

**Pull request merged**